source/sass/patterns/molecules/contextual-data.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../mixins/decorations";
+@import "../../mixins/positioning";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../settings/baselinegrid";
+@import "../../settings/color";
+
+.contextual-data {
```

Comment:
Never been a fan of the name.

Reply:
We need some suggestions then. I cannot think of anything off the top of my head at the moment apart from info-data.